modules/reference/pages/salt/salt-remote-commands.adoc

```diff
@@ -20,8 +20,8 @@ Wildcards can be used to run commands across any number of systems, so always ta
 
 On the menu:Salt[Remote Commands] page you will see two text boxes.
 The first box is for entering commands.
-The second box is for targeting clients by name, group, or by using wildcards.
+The second box is for targeting clients by the [literal]``minion_id`` value (name), or by a partial match using wildcards.
```

Comment:
Can we change this to be the field names ("in the [guimenu]``foo`` field"), rather than first and second boxes?

Reply:
As said, these boxes don't have labels.  I'm clueless...

Reply:
grrr

Reply:
> As said, these boxes don't have labels. I'm clueless...

You are right. They are `pre filled` by default, but if you delete the content you have no idea what they stand for. :disappointed: